core/templates/services/contextual/url.service.ts

```diff
@@ -182,7 +182,7 @@ export class UrlService {
     let pathname = this.getPathname();
     let queryStrings = this.getCurrentQueryString().split('=');
     if (pathname.match(/\/practice/g) && queryStrings.length === 2) {
-      return decodeURIComponent(queryStrings[1]);
+      return JSON.stringify(decodeURIComponent(queryStrings[1]).split(','));
```

Comment:
This change is done so that we can send data between backend & frontend in an unified way.
Some of the params like stringified_exploration_ids are sent in this way. It would be great if we send sub_topic_ids in the same format, so that we have a unified way to validate them in the backend.